packages/docs/examples/va-toast/Close.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <div>
+    <va-button @click="$vaToast.init({
+      message: 'Notification with onClose handler',
+      onClose: () => closeHandler
```

Comment:
That doesn't seem right, also doesn't work. `() => closeHandler()`?

Reply:
Sorry, I fixed it but forgot to push :)